docs/installation.rst

```diff
@@ -97,17 +99,12 @@ which should print out the solutions for the included test images.
 
 You can run the automated test suite with this command (from the repository root dir)::
 
+    # run all tests
     pytest
 
     # skip slow tests
     pytest -m "not slow"
 
-Using cedar-solve as a module in your repository
-
-A specific branch named `no_big_files` is available for practical inclusion as a git submodule
-in your own repository. This does not include the default database and example images, making
-it less than 1 MB. For a specific application you probably want a custom database anyway.
-
 If problems arise
 -----------------
 Please get in touch by `filing an issue <https://github.com/smroid/cedar-solve/issues>`_.
```

Comment:
While you're here, can you also add something like:

You can also join the Cedar Discord server at https://discord.gg/xbDrUyXP.